src/orion/service/testing.py

```diff
@@ -0,0 +1,140 @@
+"""Integration testing between the REST service and the client"""
```

Comment:
Could be good to add this under `orion.testing`. Perhaps in its own submodule there.